src/Sylius/Component/Promotion/Repository/PromotionCouponRepositoryInterface.php

```diff
@@ -32,4 +33,12 @@ public function createQueryBuilderByPromotionId($promotionId);
      * @return int
      */
     public function countByCodeLength($codeLength);
+
+    /**
+     * @param string $promotionCode
+     * @param string $code
+     *
+     * @return PromotionInterface
+     */
+    public function findOneByPromotionCodeAndCode($promotionCode, $code);
```

Comment:
Can't we change the order of arguments (because I didi it in the docbloc and function)?